charts/oes/templates/sapor-gate/sapor-gate-service.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq .Values.installationMode "OES-AP") (.Values.saporgate.enabled) -}}
+{{- if .Values.saporgate.enabled -}}
```

Comment:
I think we should revert this considering the use case where installation mode is set to None